README.md

```diff
@@ -1 +1,192 @@
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Dota2_Heros] <a name="about-project"></a>
+
+
+**[Dota2_Heros]** is a responsive web application that allows users to learn about Dota2 heros.Users can search for heros and learn about their abilities and stats.This project was built with React.js and Redux.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Built with react]**
+- **[Search Feature]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- Link: [Live Demo](https://dota2-heros.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A Chrome browser
+- npm
+- Node.js
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/K0ppai/dota2_heros.git
+```
+
+### Install
+
+Install the prerequisites this project using:
+
+```sh
+  cd dota2_heros
+
+  npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x  @babel/plugin-syntax-jsx@7.x  @babel/preset-react@7.x @babel/preset-react@7.x
+
+  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+  npm run start
+```
+
+### Run tests
+
+To run tests after setting up the linters, run the following command:
+
+```sh
+  npx eslint "**/*.{js,jsx}" --fix
+```
+```sh
+  npx stylelint "**/*.{css,scss}" --fix
+```
+For tests, run:
+
+```sh
+  npm run test
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Paing Soe Thu**
+
+- GitHub: [@K0ppai](https://github.com/K0ppai)
+- Twitter: [@_koppai_](https://twitter.com/_koppai_)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/paingsoe-thu)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Mobile Responsiveness]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
```

Comment:
- [ ] Please kindly fix this broken link for your README file to be engaging and professional. 👍 

```suggestion
Feel free to check the [issues page](https://github.com/K0ppai/dota2_heros/issues).
```